metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorDatabaseService.java

```diff
@@ -89,6 +96,11 @@ public void create(final ConnectorRequestContext requestContext, final DatabaseI
      */
     @Override
     public void delete(final ConnectorRequestContext requestContext, final QualifiedName name) {
+        if (this.authorizationService.unauthorized(requestContext, name, AuthorizationService.METACAT_DELETE)) {
```

Comment:
AuthorizationService should throw an exception with a generic message. 'User <username> cannot perform <action> on <name>'. Publish metric.